website/content/docs/k8s/multiport/reference/grpcroute.mdx

```diff
@@ -0,0 +1,887 @@
+  | `value`   | Specifies the number of retries to attempt.  | Integer   | None    |
+
+### `spec.rules.retries.onConditions`
+
+Specifies Envoy conditions that cause an automatic retry attempt.
+
+#### Values
+
+- Default: None
+- Data type: Map of strings
+
+### `spec.rules.retries.onConnnectFailure`
+
+Enables an automatic retry attempt when a connection failure error occurs.
+
+#### Values
+
+- Default: `false`
+- Data type: Boolean
+
+### `spec.rules.retries.onStatusCodes`
+
+Specifies the response status codes that are eligible for retry attempts.
+
+#### Values
+
+- Default: None
+- Data type: Map of integers
+
+### `spec.rules.timeouts`
+
+Specifies timeout logic when routing gRPC traffic
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `spec.rules.timeouts.idle`
+
+Specifies the total amount of time permitted for the request stream to be idle before a timeout occurs.
+
+#### Values
+
+- Default: None
+- Data type: Map that can contain the following parameters:
+
+  | Parameter | Description                                                      | Data type | Default |
+  | :-------- | :--------------------------------------------------------------- | --------- | ------- |
+  | `nanos`   | Specifies the number of nanoseconds before triggering a timeout. | Integer   | None    |
+  | `seconds` | Specifies the number of seconds before triggering a timeout.     | Integer   | None    |
+
+### `spec.rules.timeouts.request`
+
+Specifies the total amount of time spent processing the entire downstream request, including retries, before triggering a timeout.
+
+#### Values
+
+- Default: None
+- Data type: Map that can contain the following parameters:
+
+  | Parameter | Description                                                      | Data type | Default |
+  | :-------- | :--------------------------------------------------------------- | --------- | ------- |
+  | `nanos`   | Specifies the number of nanoseconds before triggering a timeout. | Integer   | None    |
+  | `seconds` | Specifies the number of seconds before triggering a timeout.     | Integer   | None    |
+
+## Examples
+
+The following examples demonstrate common GRPCRoute CRD configuration patterns for specific use cases.
+
+### Split gRPC traffic between two ports
+
+The following example splits traffic for the `api` service. GRPC traffic for services registered to the Consul catalog that are available at the `api-workload` port is split so that 50% of the traffic routes to the service at the `api-workload` port and 50% routes to the service at the `admin-workload` port.
+
+```yaml
+apiVersion: mesh.consul.hashicorp.com/v2beta1
+kind: GRPCRoute
+metadata:
+  name: api-split
+  namespace: default
+spec:
+  parentRefs:
+    - ref:
+        type: 
+          group: catalog
+          groupVersion: v2beta1
+          kind: Service
+        name: api
+      # Note that right now, we must target the workload port,
+      # not the service port. 
```

Comment:
```suggestion
      # not the service port. In a future release, this will be the service port name.
```
We might want to update similar comments with this text.